symplyphysics/laws/waves/phase_velocity_from_wavelength_and_time_period.py

```diff
@@ -0,0 +1,44 @@
+from sympy import Eq
```

Comment:
We already have this: https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/waves/wavelength_from_wave_speed_and_period.py

Reply:
Oh, I overlooked